afs-ws/afs-ws-client/src/main/java/com/powsybl/afs/ws/client/RemoteAppFileSystemProvider.java

```diff
@@ -47,13 +48,14 @@ public RemoteAppFileSystemProvider(Supplier<Optional<RemoteServiceConfig>> confi
     @Override
     public List<AppFileSystem> getFileSystems(AppFileSystemProviderContext context) {
         Objects.requireNonNull(context);
-        Optional<RemoteServiceConfig> config = configSupplier.get();
-        if (config.isPresent()) {
-            URI uri = config.get().getRestUri();
+        RemoteServiceConfig config = configSupplier.get().orElse(null);
+        if (config != null) {
+            URI uri = config.getRestUri();
```

Comment:
why remove the nicer optional null handling ?

Reply:
Ah, matter of taste I guess :) I find that when you use an optional just to check "ifPresent" and then use "get", it's uglier/heavier because you need either to introduce an additional variable for the unwrapped object, or call get() multiple times.

No problem to change it back though!